expressEngine/lib/responseHandler/responseHandler.js

```diff
@@ -0,0 +1,17 @@
+import { error as commonError, success } from '../../cms/common';
+
+class responseHandler {
+  success = ({ data = null, message = success.defaultSuccessMsg }) => ({ data, message });
+
+  error = ({ error = commonError.undefinedError, status = 500 }) => ({ error, status });
+
+  response = (data) => {
```

Comment:
@chiragSuccessive Please use response object here so that you can send the necessary response.

Reply:
ok